tickets-test/src/main/java/publication/test/TicketTestUtils.java

```diff
@@ -0,0 +1,88 @@
+package publication.test;
```

Comment:
Terrible choice of package name. no.unit.nva.publication.ticket.test

Reply:
Endrer til no.unit.nva.publication.ticket.test